server/src/portfolio/cron.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import cron from 'node-cron'
+import { pgPool } from './db/dbClient.js';
+import { updatePortfolioValue } from './portfolio.js';
+
+
+
```

Comment:
This file doesn't seem to be imported / used in server.js.

Maybe make an exported function that calls cron.schedule, and runs this fn in server.js?